build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -912,6 +953,9 @@ message OutputFile {
   // message. The server MAY omit inlining, even if requested, and MUST do so if inlining
   // would cause the response to exceed message size limits.
   bytes contents = 5;
+
+  // The available properties of the file
```

Comment:
Here and in other output node_properties: "The supported node properties of the <thing>, if requested by the Action."